RBGS.f

```diff
@@ -394,15 +394,16 @@ subroutine rbgs(
             print *, info
           endif
 c
-          un_data_0(i0,i1) = b(1);
-          un_data_0(i0+1,i1) = b(2);
-          un_data_1(i0,i1) = b(3);
-          un_data_1(i0,i1+1) = b(4);
-          us_data_0(i0,i1) = b(5);
-          us_data_0(i0+1,i1) = b(6);
-          us_data_1(i0,i1) = b(7);
-          us_data_1(i0,i1+1) = b(8);
-          p_data(i0,i1) = b(9);
+          w = 0.75d0  
```

Comment:
Instead of setting `w` here, make it an input to the function. Then make `w` a member variable of the FAC operator class which gets read in via the input file (set in the constructor). That way we can change the value of `w` without having to recompile.

Reply:
Done.